packages/grafana-runtime/src/analytics/utils.ts

```diff
@@ -43,14 +43,17 @@ export const reportPageview = () => {
  *
  * @public
  */
-export const reportInteraction = (interactionName: string, properties?: Record<string, unknown>) => {
-  getEchoSrv().addEvent<InteractionEchoEvent>({
-    type: EchoEventType.Interaction,
-    payload: {
-      interactionName,
-      properties,
+export const reportInteraction = (interactionName: string, properties?: Record<string, unknown>, meta?: {}) => {
+  getEchoSrv().addEvent<InteractionEchoEvent>(
+    {
+      type: EchoEventType.Interaction,
+      payload: {
+        interactionName,
+        properties,
+      },
```

Comment:
Why do we need an extra prop `meta`? Can we just add it to the `properties`?

Reply:
Yeah, you might be right setting this as a property is the way to go. Pushed an update